test.js

```diff
@@ -38,6 +38,7 @@ assert.equal('five', five.english(), 'A english five should be five');
 assert.equal('kvin', five.esperanto(), 'An esperanto five should be kvin');
 assert.equal('viis', five.estonian(), 'An estonian five should be viis');
 assert.equal('viisi', five.finnish(), 'A finnish five should be viisi');
+assert.equal('viisi', five.flatgerman(), 'A flatgerman five should be fief');
```

Comment:
The test is failing see: https://travis-ci.org/jackdcrawford/five/builds/290141384
`flatgerman`should return `fief`instead of `viisi`, right?

Reply:
Yea, thats what i've expected...

Reply:
assert.equal('viisi', .... 'should be fief');